contrib/gitian/README.md

```diff
@@ -201,10 +234,11 @@ Make a pull request (both the `.assert` and `.assert.sig` files) to the
 [monero-project/gitian.sigs](https://github.com/monero-project/gitian.sigs/) repository:
 
 ```bash
+cd gitian.sigs
 git checkout -b $VERSION
 # add your assert and sig files...
 git commit -S -a -m "Add $GH_USER $VERSION"
-git push --set-upstream $GH_USER $VERSION
+git push --set-upstream origin $VERSION
```

Comment:
With the repo setup we discussed here, this line should be left as the original text.